### PR TITLE
chore(C): add prefix to postfix in stack

### PR DESCRIPTION
**Pull request by RohanSasne**
**PR Checklist:**
- [ ] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [ ] My addition is on refer on the language README.md file
- [ ] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi @roohan25,
Please rename the file from `Prefix-to-postfix-using-stack1.c` to `Prefix-to-postfix-using-stack.c`

**Comment by roohan25** [comment]
The requested changes have been updated, Please check once and if any query exist, do reach out , will be happy to resolve the issue .

File: algorithms/C/stacks/Prefix-to-postfix-using-stack.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    //scanning from right to left
+    for(i = l - 1; i >= 0; i--)
+    {
+        //checking if the symbol is an operator
+        if (is_operator(exp[i]))
+        {
+            //popping two operands from stack
+            strcpy(op1, pop()) ;
+            strcpy(op2, pop()) ;;
+            //concating the operands and operator
```
Comment (mohitchakraverty) [inline algorithms/C/stacks/Prefix-to-postfix-using-stack.c:47]:
```suggestion
            //concatenating the operands and operator
```

**Review by mohitchakraverty — CHANGES_REQUESTED** [review]
Please make fix the following typo and also update your respective README
In your case it will be - algorithms/C/README.md

**Pull request closed without merging**